Tesla.css

```diff
@@ -0,0 +1,477 @@
+* {
+  margin: 0;
+}
+
+body {
+  max-width: 1920px;
+  margin: 0 auto;
+  font-family: Montserrat, sans-serif;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.header {
+  position: relative;
+}
+
+.logo {
+  width: 166px;
+  height: 51px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
+}
+
+.nav {
+  width: 93%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  margin-top: 24px;
+}
+
+.nav__link {
+  font-size: 15px;
+  white-space: nowrap;
+  margin: 0 10px;
+}
+
+.nav__model {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 37%;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
+
+.nav__input {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 14%;
+}
+
+.nav__link:hover {
+  color: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)